lib/Entity/User.php

```diff
@@ -805,6 +805,12 @@ public function reassignAllTo($user)
             'oldUserId' => $this->userId
         ]);
 
+        // Reassign oAuth Clients
```

Comment:
This is interesting, although I am not convinced that this does not present a security problem (essentially granting API access to someone else by virtue of deleting a user).

It might be better here for us to delete the oauth_client - what do you think?